etl/data_helpers/geo.py

```diff
@@ -740,27 +740,29 @@ def add_gdp_to_table(
 
     Returns
     -------
-    tb_with_gdo : Table
+    tb_with_gdp : Table
         Original table after adding a column with GDP values.
 
     """
     tb_with_gdp = tb.copy()
 
     # Read main table from GDP dataset.
-    tb_gdp = ds_gdp["maddison_gdp"].reset_index()
-
-    # Add metadata sources and licenses to the main GDP variable.
-    tb_gdp["gdp"].metadata.sources = ds_gdp.metadata.sources
-    tb_gdp["gdp"].metadata.licenses = ds_gdp.metadata.licenses
+    error = (
```

Comment:
In the future we can maybe integrate the World Bank and the Penn World Table estimates of GDP, to cover all the cases we'd need